src/application/application.js

```diff
@@ -16,32 +12,5 @@ export default Application.extend({
     this.$body = $(document.body);
     this.layout = new LayoutView();
     this.layout.render();
-
-    this.listenTo(routerChannel, {
-      'before:enter:route' : this.onBeforeEnterRoute,
-      'enter:route'        : this.onEnterRoute,
-      'error:route'        : this.onErrorRoute
-    });
-  },
-
-  onBeforeEnterRoute() {
-    this.transitioning = true;
-    // Don't show for synchronous route changes
-    _.defer(() => {
-      if (this.transitioning) {
-        nprogress.start();
-      }
-    });
-  },
-
-  onEnterRoute() {
-    this.transitioning = false;
-    this.$body.scrollTop(0);
-    nprogress.done();
-  },
-
-  onErrorRoute() {
-    this.transitioning = false;
-    nprogress.done(true);
```

Comment:
This shouldn't be part of this PR, only in https://github.com/thejameskyle/marionette-wires/pull/70